Libs/MRML/Core/vtkMRMLSliceNode.cxx

```diff
@@ -84,6 +84,10 @@ vtkMRMLSliceNode::vtkMRMLSliceNode()
 
   this->SliceResolutionMode = vtkMRMLSliceNode::SliceResolutionMatch2DView;
 
+  this->SlabReconstructionVisibility = false;
+  this->SlabReconstructionType = vtkMRMLSliceNode::SliceReconstructionMax;
+  this->SlabThickness = 1;
```

Comment:
I am wondering if this should renamed to `SlabThicknessInMm` ?  If not, add a comment like